### PR TITLE
Exclude venv from flake8 linting

### PR DESCRIPTION
**Pull request by leo-gan**
Issue: flake8 processes the venv dir.
Change: added `exclude = venv` to flake8 conf

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -3,3 +3,5 @@ max-line-length = 120
 ignore = E203,E501,W503
 per-file-ignores =
   __init__.py:F401
+exclude =
+    venv
```
Comment (patricklabatut) [inline setup.cfg:7]: This is only needed because of your specific workflow that uses a virtualenv?

Comment (leo-gan) [reply setup.cfg:7]: It is recommended to use a virtual environment for a project. There could be different configurations for this virtenv. I've added here a typical one. It also works as a placeholder to help with adding other specific configurations.

Comment (patricklabatut) [reply setup.cfg:7]: > It is recommended to use a virtual environment for a project.

Not everyone is using specifically virtual environments to isolate installations, e.g. we are using conda environment instead. And the location is pretty specific although I supposed it's the commonly used naming convention.

**Review by patricklabatut — APPROVED** [review]

**Comment by patricklabatut** [comment]
Thanks for the suggestion @schmidt-ai, taking notes and will evaluate `ruff` but for now, let's just proceed with `flake8`.

**Pull request merged**